src/ResourceManager/Relay/Commands.Relay/packages.config

```diff
@@ -1,8 +1,16 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Microsoft.Azure.Management.Authorization" version="1.0.2" targetFramework="net45" />
-  <package id="Microsoft.Azure.Management.Relay" version="0.0.1-preview" targetFramework="net45" />
+  <package id="Microsoft.Azure.Management.Relay" version="2.0.0" targetFramework="net452" />
   <package id="Microsoft.Azure.Management.Resources" version="2.20.0-preview" targetFramework="net45" />
   <package id="Microsoft.Azure.Management.Storage" version="3.0.0" targetFramework="net45" />
   <package id="Microsoft.Azure.Management.Websites" version="1.2.0-preview" targetFramework="net45" />
+  <package id="Microsoft.Bcl" version="1.1.9" targetFramework="net45" />
```

Comment:
@v-Ajnava similar comment to the one made in the csproj file. The `packages.config` file should only have a reference on the management packages you need.